src/wallet/rpc/coins.cpp

```diff
@@ -356,16 +356,12 @@ RPCHelpMan lockunspent()
         outputs.push_back(outpt);
```

Comment:
Unrelated: I find the `lockunspent` RPC request params unusual though. The second arg is called `transactions` but expects transaction outputs. The RPC is called `lockunspent` but the first arg is called `unlock` that leads me to straightaway do mental negation calculation while calling the RPC that seems undesirable to me.

Reply:
It is a bit weird, but changing the interface is probably not going to happen as it would be a breaking change.